vns3/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - Nomad
```

Comment:
Hmm not sure why this is listed as Nomad

Reply:
Nor am I - not even sure what it is referring to. Don't think I put it there. Is it an issue?

Reply:
```suggestion
# CHANGELOG - VNS3
```
Non blocking nit. I guess just some good housekeeping.